testing/install_packages_remote_pkgspec_test.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+substitutions:
+  '_TEST': 'install_packages_remote_pkgspec_test'
+  '_INPUT_IMAGE': 'cos-85-13310-1260-8'
+  '_INPUT_PROJECT': 'cos-cloud'
+  '_INPUT_PKGSPEC_FILE': 'kubernetes_pkg_spec.tar.gz'
+  '_INPUT_PKGSPEC': 'testing/install_packages_remote_pkgspec_test/kubernetes_pkg_spec.tar.gz'
+
+steps:
+- name: 'gcr.io/cloud-builders/bazel'
+  args: ['run', '--spawn_strategy=standalone', ':cos_customizer', '--', '--norun']
+- name: 'gcr.io/cloud-builders/gsutil'
+  args: [ "-m", "cp", "${_INPUT_PKGSPEC}", "gs://${PROJECT_ID}_cloudbuild/" ]
+- name: 'bazel:cos_customizer'
+  args: ['start-image-build',
+         '-build-context=testing/${_TEST}',
+         '-image-name=${_INPUT_IMAGE}',
+         '-image-project=${_INPUT_PROJECT}',
+         '-gcs-bucket=${PROJECT_ID}_cloudbuild',
+         '-gcs-workdir=customizer-$BUILD_ID']
+- name: 'bazel:cos_customizer'
+  args: ['anthos-installer-install',
+         '-pkgspec-url=gs://${PROJECT_ID}_cloudbuild/${_INPUT_PKGSPEC_FILE}']
+- name: 'gcr.io/cloud-builders/gsutil'
+  args: [ "rm", "gs://${PROJECT_ID}_cloudbuild/${_INPUT_PKGSPEC_FILE}" ]
```

Comment:
How does this work? It looks like we're deleting the file before it gets downloaded (I think it gets downloaded during the build in finish-image-build?)

Reply:
Yes thats true. I added the rm command at the last minute and missed to validate it.